server/src/main/java/org/elasticsearch/search/aggregations/bucket/histogram/VariableWidthHistogramAggregationBuilder.java

```diff
@@ -136,6 +136,11 @@ public BucketCardinality bucketCardinality() {
         return BucketCardinality.MANY;
     }
 
+    @Override
+    public boolean supportsConcurrentExecution() {
+        return false;
```

Comment:
for future reference maybe it would be a good idea to have a small comment that says why so that it does not have to be traced back to the PR and the explanation from the PR?